### PR TITLE
Oxid integration.

### PR DESCRIPTION
**Pull request by GrandLTU**

### CODE REVIEW

**Comment by GrandLTU** [comment]
Waiting for https://github.com/ongr-io/OXIDConnectorBundle/pull/13

File: src/ONGR/DemoOXIDBundle/Document/AttributeObject.php [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * (c) NFQ Technologies UAB <info@nfq.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ONGR\DemoOXIDBundle\Document;
+
+use ONGR\ElasticsearchBundle\Annotation as ES;
+use ONGR\OXIDConnectorBundle\Document\AttributeObject as ParentObject;
```
Comment (lmikelionis) [inline src/ONGR/DemoOXIDBundle/Document/AttributeObject.php]: Why do you use general namig?

Comment (tautrimas) [inline src/ONGR/DemoOXIDBundle/Document/AttributeObject.php]: - [x] Because of name collision.

File: vagrant/manifests/default.pp [hunk 2]
```diff
@@ -157,7 +157,7 @@
   changes => [
   "set PHP/date.timezone Europe/Vilnius",
   "set XDEBUG/xdebug.default_enable 1",
-  "set XDEBUG/xdebug.max_nesting_level 250",
+  "set XDEBUG/xdebug.max_nesting_level 2500",
```
Comment (mindaugasbarysas) [inline vagrant/manifests/default.pp]: This should not be necessary any more.

**Pull request merged**